### PR TITLE
emilua: 0.6.0 -> 0.7.2

### PR DESCRIPTION
**Pull request by manipuladordedados**
## Description of changes

version update.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by manipuladordedados** [comment]
Sorry to open another, my git config fuc#$5 the things up.

File: pkgs/development/interpreters/emilua/default.nix [hunk 1]
```diff
@@ -90,7 +93,15 @@ stdenv.mkDerivation rec {
...
 
-    substituteInPlace src/emilua_gperf.awk  --replace '#!/usr/bin/env -S gawk --file' '#!${gawk}/bin/gawk -f'
+    patchShebangs src/emilua_gperf.awk --interpreter '${lib.getExe gawk} -f'
+  '';
+
+  doCheck = true;
+
+  # Skipped test: libpsx
+  # Known issue with no-new-privs disabled in the Nix build environment.
+  checkPhase = ''
```
Comment (lucasew) [inline pkgs/development/interpreters/emilua/default.nix:103]:
```
runHook preTest

meson test ...

runHook postTest

```

**Comment by lucasew** [comment]
Result of `nixpkgs-review pr 308155` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>emilua</li>
  </ul>
</details>

**Comment by lucasew** [comment]
Result of `nixpkgs-review pr 308155` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>emilua</li>
  </ul>
</details>

**Review by lucasew — APPROVED** [review]

File: pkgs/development/interpreters/emilua/default.nix [hunk 2]
```diff
@@ -90,7 +93,17 @@ stdenv.mkDerivation rec {
...
+  runHook preTest
+    meson test --print-errorlogs --no-suite libpsx
+  runHook postTest
```
Comment (Artturin) [inline pkgs/development/interpreters/emilua/default.nix:106]:
```suggestion
  runHook preCheck
    meson test --print-errorlogs --no-suite libpsx
  runHook postCheck
```

File: pkgs/development/interpreters/emilua/default.nix [hunk 3]
```diff
@@ -90,7 +93,17 @@ stdenv.mkDerivation rec {
...
+  '';
+
+  doCheck = true;
+
+  # Skipped test: libpsx
+  # Known issue with no-new-privs disabled in the Nix build environment.
+  checkPhase = ''
+  runHook preCheck
+    meson test --print-errorlogs --no-suite libpsx
+  runHook postCheck
```
Comment (Artturin) [inline pkgs/development/interpreters/emilua/default.nix:106]:
```suggestion
    runHook preCheck
    meson test --print-errorlogs --no-suite libpsx
    runHook postCheck
```

**Pull request merged**